server/src/ocv.rs

```diff
@@ -43,6 +43,116 @@ impl Ocv {
     Ok(ProposalResponse { proposal, votes })
   }
 
+  pub async fn proposal_consideration(&self, id: usize, start_time: i64, end_time: i64, ledger_hash: Option<String>) -> Result<GetMinaProposalConsiderationResponse> {
+    let proposal_key = "MEF".to_string() + &id.to_string();
+    let votes = if let Some(cached_votes) = self.caches.votes.get(&proposal_key).await {
+      cached_votes.to_vec()
+    } else {
+      let transactions = self.archive.fetch_transactions(start_time, end_time)?;
+
+      let chain_tip = self.archive.fetch_chain_tip()?;
+      let votes = Wrapper(transactions.into_iter().map(std::convert::Into::into).collect())
+      .process_mep(&id.to_string(), chain_tip)
```

Comment:
according to clippy borrow is not needed here.
```suggestion
      .process_mep(id.to_string(), chain_tip)
```

Reply:
fixed